src/nl/blog/2024/12/mijn-favoriete-tools-als-developer.md

```diff
@@ -0,0 +1,82 @@
+
+Als front-enddeveloper met een paar jaar ervaring, vind ik het altijd interessant om te weten hoe andere developers hun werk doen, welke tools gebruiken zij en hoe worden die gebruikt. Daarom zet ik in deze blog de tools die ik gebruik om anderen te inspireren.
+
+## Hoe ik mijn computer gebruik
+
+Ik doe eigenlijk alles op mijn eigen laptop die op Windows 11 draait, mijn werk dus ook. Dat laatste klinkt misschien wat apart, maar eigenlijk vind ik het wel fijn op dit moment op gewoon op mijn eigen laptop te werken. Ik kan alles makkelijk terugvinden en kan gewoon overal bij. 
+
+Windows 11 is prettig voor mij onderandere vanwege de toegankelijkheidsinstellingen, zoals Plaktoetsen, die ik gebruik omdat ik blind typ met één hand. Hierdoor hoef ik toetsen niet tegelijk in te drukken. Ik pas ook regelmatig de toetsenbordinstellingen aan met MyAsus of Logi Options+ om functietoetsen snel te wisselen. Jammer genoeg werkt de Fn-toets niet samen met Plaktoetsen.
+
+Ik werk als developer voornamelijk met WordPress. Ik bouw de websites voor onze klanten en onderhoud en help onze klanten als zij vragen of problemen hebben bij het gebruik van onze WordPress websites.
+
+## Mijn tools op een rij
+
+1. browsers: Chrome, Firefox en Microsoft Edge
+De belangrijkste tool die ik gebruik bij mijn werk is Chrome. Ik weet dat niet iedereen een grote fan is van Chrome, maar ik vind het wel het beste werken met Google workspace diensten die wij op het werk gebruiken en bovendien vind ik de developer tools prettig in gebruik. Naast Chrome Gebruik ik ook Firefox en Microsoft Edge om te testen hoe een website in een andere browser werkt en hoe een website werkt als ik ben ingelogd met een andere userrole.
+Belangrijke extensies die ik gebruik zijn WAVE om te testen op toegankelijkheid, Wappalyzer om te kijken welke technologieen een website gebruikt. ColorZilla om hexcodes op websites te achterhalen en Hoog contrast om te kijken hoe websites eruit zien in hoog contrast.En natuurlijk een passwordmanager.
+
+2. ClickUp
+Als tool voor projectmanagement gebruiken we ClickUp. Ik vond het in het begin behoorlijk overweldigend, maar het is erg handig omdat je er allerlei project gerelateerde informatie in kwijt kunt, zoals wat er moet gebeuren en wie dat moet doen, hoelang we eraan hebben gewerkt en nog veel meer!
+
+3. Figma
+De designs voor de websites worden door onze designer gemaakt in Figma. Vervolgens kijk ik in Figma hoe de website eruit moet komen te zien. 
+
+4. Slack
+Ik werk zelf voornamelijk vanuit huis en om om te communiceren met collega’s gebruiken we voornamelijk Slack. Niet alleen voor werk gebruik ik Slack, maar ook de ben ik regelmatig te vinden in de [Fronteers Slack](https://fronteersnl.slack.com/) en in de Slack workspace van WordPress Nederland. 
+
+![Screenshot van Visual Studio Code waarin het bestand functions.php is geopend.](_img\blog\2024\12\mijn-favoriete-tools-als-front-end-developer\screenshot-vs-code.jpg)
```

Comment:
Met correcte slashes (ook een aan het begin) werken de images beter:

```suggestion
![Screenshot van Visual Studio Code waarin het bestand functions.php is geopend.](/_img/blog/2024/12/mijn-favoriete-tools-als-front-end-developer\screenshot-vs-code.jpg)
```

Reply:
Lokaal worden de abeeldingen nog niet goed geladen, Dat ligt denk ik aan het file path? 
<img width="654" alt="image" src="https://github.com/user-attachments/assets/6ed15caf-00df-490d-9a17-9094a3ca7890" />

Reply:
Het werkt nu ik `_img/blog/2024/12/mijn-favoriete-tools-als-front-end-developer/screenshot-vs-code.jpg
` heb veranderd in `/_img/blog/2024/12/mijn-favoriete-tools-als-front-end-developer/screenshot-vs-code.jpg
`